openapi/tags/uploads.yaml

```diff
@@ -131,6 +132,17 @@ paths:
       description: |-
         Fetches information about the file at `path` within the upload.
         It is not currently possible to retrieve the file contents.
+      parameters:
+        - name: marker
+          in: query
+          description: |-
+            The name of a hypothetical file indicating where to start when listing the children of a folder.
```

Comment:
> hypothetical

seems out of place here maybe: _A file name, that indicates where to start when listing the children of a folder._

Reply:
Updated